nimble/host/store/config/src/ble_store_config.c

```diff
@@ -23,7 +23,9 @@
 #include "sysinit/sysinit.h"
 #include "syscfg/syscfg.h"
 #include "host/ble_hs.h"
+#ifdef BLE_STORE_CONFIG_PERSIST
```

Comment:
That was also my first solution then I found out that it seems that content of included file is not used at all, hence #889

Reply:
Ok, so as your PR has better solution I think this one can be closed.